modules/ocf/manifests/packages/firefox.pp

```diff
@@ -3,14 +3,20 @@
     stage => first,
   }
 
-  $firefox_pkg = $::lsbdistcodename ? {
-    'jessie' => 'firefox',
-    default  => 'firefox-esr',
+  if $::lsbdistcodename == 'jessie' {
```

Comment:
Good catch. We can probably do this with `aptitude`, `apt-show-versions`, or `apt-cache policy`.